packages/autocannon-ui-frontend/src/components/ProgressBar.jsx

```diff
@@ -5,16 +5,19 @@ import Typography from '@material-ui/core/Typography'
 import Box from '@material-ui/core/Box'
 import T from 'prop-types'
 
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   root: {
     width: '70%',
-    padding: '30px'
+    padding: theme.spacing(3)
   },
   bar: {
-    height: '10px',
-    borderRadius: '5px'
+    height: theme.spacing(1),
+    borderRadius: theme.spacing(1)
```

Comment:
fine to use spacing too, but note that the preference to use spacing is for.. spacing, meaning padding and margin. element size and shapes don't necessarily have to use multiples of the base unit